source/manual/ruby.html.md

```diff
@@ -32,17 +32,33 @@ We set up rbenv differently depending on what's going on:
 
 ## Add a new Ruby version in puppet
 
-Build a package using [fpm](debian-packaging.html#fpm) and make it
-available using our aptly repository.
+You will need to build a new [fpm](debian-packaging.html#fpm) package with the new Ruby version.
+This package can then be copied to Aptly machine, and the new version added to puppet.
 
+### Building the fpm package.
+
+- Add a new recipe for the ruby version in [Packager](packager).
+The folder name will be the Ruby version, and contain a `recipe.rb` file. See previous entries for examples.
+The recipe will require the SHA256 of the version's `tar.gz` file.
```

Comment:
It might be good to expand on how we do this:
 - from where we take the `tar.gz` file
 - how to create the SHA256 from a file (ruby command or link to a website)

Reply:
good ideas, thanks